tests/integration/src/jtag/test_lc_transitions.rs

```diff
@@ -6,29 +6,32 @@ mod test {
 
     use caliptra_hw_model::openocd::openocd_jtag_tap::{JtagParams, JtagTap};
     use caliptra_hw_model::Fuses;
+    use caliptra_hw_model::DEFAULT_LIFECYCLE_RAW_TOKEN;
     use mcu_builder::FirmwareBinaries;
-    use mcu_hw_model::lcc::{lc_transition, read_lc_state};
+    use mcu_hw_model::lcc::{lc_token_to_words, lc_transition, read_lc_state};
     use mcu_hw_model::{DefaultHwModel, InitParams, McuHwModel};
     use mcu_rom_common::LifecycleControllerState;
 
-    #[test]
-    fn test_raw_unlock() {
+    fn ss_setup(initial_lc_state: Option<LifecycleControllerState>) -> Box<DefaultHwModel> {
```

Comment:
You shouldn't need to use box here, if you are using `DefaultHwModel`. If you were using `McuHwModel` you would though

Reply:
fixed